QuantumLink/Common/Networking/Protocol/Packet.cs

```diff
@@ -2,50 +2,50 @@
 using System.Net.Sockets;
 using System.Text;
 
-namespace QuantumLink.networking.protocol
+namespace QuantumLink.Common.Networking.Protocol
 {
     public abstract class Packet
     {
         public readonly byte Opcode;
-        protected readonly Stream stream;
+        protected readonly Stream Stream;
 
         public Packet(byte opcode, Stream stream)
         {
             this.Opcode = opcode;
-            this.stream = stream;
+            this.Stream = stream;
         }
     }
 
     public class OutboundPacket : Packet
     {
-        protected readonly BinaryWriter writer;

```

Comment:
Why use get. Readonly is more use-case specific, and accurate

Reply:
Properties are better